local/sdk-sriov/pkg/tools/socket_path.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package tools provide different tools
+package tools
+
+import (
+	"net"
+	"os"
+
+	"github.com/pkg/errors"
+)
+
+// SocketPath is a unix socket file path.
```

Comment:
It looks like we've moved this file from monorepo and  I think we should add Cisco copyright.

Reply:
Thx, fixed